exec/java-exec/src/main/java/org/apache/drill/exec/testing/ExecutionControls.java

```diff
@@ -80,7 +80,7 @@
      * @param ttl  the number of queries for which this option should be valid
      */
     public ControlsOptionValidator(final String name, final String def, final int ttl) {
-      super(name, OptionValue.Kind.STRING, OptionValue.createString(OptionType.SESSION, name, def));
+      super(name, OptionValue.Kind.STRING, OptionValue.createString(OptionType.SYSTEM, name, def));
```

Comment:
can you add a comment explaining why this needs to be a _SYSTEM_ rather than a _SESSION_ option ? someone else might revert this change inadvertently.